README.md

```diff
@@ -128,7 +128,7 @@ like `user`, `password` and `db` etc.
 To use this connector with SQLAlchemy, use the `creator` argument for `sqlalchemy.create_engine`:
 
 ```python
-from google.cloud.alloydb.connector import Connector
+from google.cloud.alloydb_connector import Connector
```

Comment:
Let's add a note in the README somewhere about the package name. The gist of the note should say:

- There are two ways to import the alloydb connector.
- The preferred way to alloydb_connector to avoid namespace collisions.
- Meanwhile, alloydb.connector will continue to work forever.